build.sbt

```diff
@@ -28,7 +28,7 @@ lazy val root = (project in file("."))
         "com.dwolla" %% "scala-cloudformation-custom-resource" % "4.0.0-M2",
         "com.dwolla" %% "fs2-aws" % "2.0.0-M5",
         "io.circe" %% "circe-fs2" % "0.12.0",
-        "com.dwolla" %% "cloudflare-api-client" % "4.0.0-M10",
+        "com.dwolla" %% "cloudflare-api-client" % "4.0.0-M13-SNAPSHOT",
```

Comment:
This will be updated once https://github.com/Dwolla/scala-cloudflare/pull/39 has been merged in.

Reply:
This should now be updated.